### PR TITLE
German README.md

### PR DESCRIPTION
**Pull request by FierceSkit**
Added a German README.md, since there'd been important changes lately.
Please Read & Review - any Suggestions, aswell as Corrections welcome.

### CODE REVIEW

**Comment by patricklodder** [comment]
Added `help wanted` -> needs a native speaker to review.

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:35]:
```suggestion
Dogecoin Core verwendet den Port 22556 als Standardport zur Kommunikation
```

Changed "Standart-Port" to "Standardport" and "Kommunkation" to "Kommunikation" for correct spelling.

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:38]:
```suggestion
Zusätzlich kann ein JSON-RPC-Port geöffnet werden, der den Port 22555 als Standardport für Mainnet-Nodes verwendet.
```

Added hyphens for "JSON-RPC-Port" and "Mainnet-Nodes" to match German compound noun formatting.

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:40]:
```suggestion
**Es wird dringend davon abgeraten, RPC-Ports im öffentlichen Internet sichtbar zu machen!**
```

Changed "STRENGSTENS" to "dringend" for a more typical expression and corrected "Sichtbar" to "sichtbar."

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:49]:
```suggestion
Dogecoin Core ist eine Open-Source-Software und wird von der Community entwickelt.
```

Added a hyphen to "Open-Source-Software" for correct compound noun formatting.

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:50]:
```suggestion
Der Entwicklungsprozess ist transparent und öffentlich einsehbar; jeder kann ihn sehen, darüber diskutieren und daran teilhaben!
```

Lowercased "Einsehbar" to "einsehbar" and "Jeder" to "jeder" for consistency in capitalization.

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:62]:
```suggestion
Die Vergabe der Versionsnummern erfolgt nach dem Prinzip von ```major.minor.patch```.
```

Corrected "efolgt" to "erfolgt".

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:65]:
```suggestion
Es gibt 4 Arten von Branches in diesem Repository:
```

Removed quotation marks around "Branches" for consistency.

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:80]:
```suggestion
Wenn Sie an der Mithilfe oder der Entwicklung von Dogecoin Core interessiert sind, sehen Sie sich bitte den [Beitragsleitfaden](CONTRIBUTING.md) an.
```

Added a comma and lowercased "Bitte" for correct punctuation and consistency.

File: doc/intl/de_GER/README.md
Comment (loom4k) [inline doc/intl/de_GER/README.md:87]:
```suggestion
Vielleicht findet sich ja auch jemand, der Ihnen bei Ihren Projekten behilflich sein kann.
```

Lowercased "Jemand" and changed "ihren" to "Ihren" to match the capitalization and case correctly.

**Review by loom4k — CHANGES_REQUESTED** [review]
I would change a couple of things, mainly spelling mistakes and incorrect capitalization. Overall a really good translation.

File: doc/intl/de_GER/README.md
Comment (loom4k) [reply doc/intl/de_GER/README.md:65]: Sounds good to me. Let's keep it as you had it.

File: doc/intl/de_GER/README.md
Comment (loom4k) [reply doc/intl/de_GER/README.md:35]: I made a typo in my suggestion and forgot to add the back braces. That's on me. Definitely keep it as `22556`.

File: doc/intl/de_GER/README.md
Comment (loom4k) [reply doc/intl/de_GER/README.md:38]: Again, typo on my side for the back quotation marks.

File: doc/intl/de_GER/README.md
Comment (loom4k) [reply doc/intl/de_GER/README.md:40]: No problem with that. Let's keep it as you had it 👍

**Comment by loom4k** [comment]
Looks all good to me.

**Review by loom4k — APPROVED** [review]
Looks all good to me.

**Comment by loom4k** [comment]
Absolutely. I will do this as soon as I get home. Might be a couple of hours however. Will leave a CR ASAP.

**Review by loom4k — APPROVED** [review]
This looks all good on my side. Great translation work by @FierceSkit! @patricklodder I'd say that's a wrap. This would be ready to merge when you are 😉

**Comment by loom4k** [comment]
No problem. I didn't think of leaving it open but it's actually a great idea. I'm so used of being in a rush. Let's see what others think!

File: doc/intl/de_GER/README.md
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:86]:
```suggestion
Herzlich sind sie Eingeladen sich umzusehen um neue Leute kennenzulernen, die neuesten Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
```
Changed singular to plural.

File: doc/intl/de_GER/README.md
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:86]:
```suggestion
Herzlich sind Sie eingeladen sich umzusehen um neue Leute kennenzulernen, die neuesten Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
```
Spelling.

**Review by DGalapagos8 — APPROVED** [review]
looks good to me.

File: doc/intl/de_GER/README.md
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:69]:
```suggestion
- **development:** Instabil, enthält neuen Code für kommende Versionen. Format: ```<version>-dev```
```

**Review by DGalapagos8 — CHANGES_REQUESTED** [review]

File: doc/intl/de_GER/README.md
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:36]:
```suggestion
mit dem Peer-to-Peer Netzwerk und um die "Mainnet" Blockchain zu synchronisieren.
```

File: doc/intl/de_GER/README.md
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:35]:
```suggestion
Dogecoin Core verwendet den Port `22556` als Standart-Port zur Kommunikation
```

**Review by DGalapagos8 — CHANGES_REQUESTED** [review]
Those are the last things I found. spelling and grammer.

**Review by DGalapagos8 — APPROVED** [review]
I totally approve this.

**Comment by patricklodder** [comment]
I'll squash this for you.

**Review by patricklodder — APPROVED** [review]
utACK. This still needs integration with the index in `doc/intl/README.md`, but that could be done separately too.

Since I squashed this, needs a second ACK from another maintainer.

**Review by michilumin — APPROVED** [review]
Drawing on my German grom growing up in a weird school where it was spoken primarily -- LGTM. :)

**Pull request merged**
